### DISCUSSION TITLE
Section 3 => Error in Script.sol

### DISCUSSION THREAD

**Discussion by vinaybh01** [Cyfrin/security-and-auditing-full-course-s23#248] [Q&A] [answered]
I am getting error in Script.sol while deploying the contract, i am not able to see the files in the forge-std floder so what should i do now? can any please help me with this

![image](https://github.com/user-attachments/assets/87d8d37d-4038-4727-bf50-0369e57f9546)

**Comment by vinaybh01** [comment, marked as answer]
i have not installed the dependency now i remembered 

` forge install `

**Reply by EngrPips** [reply]
I am glad you figured this out.
